src/components/Navigation/Sections/DescribedSection.js

```diff
@@ -31,7 +32,10 @@ class DescribedSection extends Component {
       applyCurationSuggestion
     } = this.props
     const definition = Contribution.foldFacets(rawDefinition, activeFacets)
-    const toolList = get(definition.described, 'tools', []).map(tool =>
+
+    // trim tool text if it starts with curation, as what follows is a long hash
+    // e.g. curation/1032ce558db47eee59b75d84132f5fbe6e19efbb
+    definition.described.tools = get(definition.described, 'tools', []).map(tool =>
       tool.startsWith('curation') ? tool.slice(0, 16) : tool
     )
```

Comment:
I'm a little concerned about mutating the `definition` object here and the ripple effects this might have through the app.

Can the ListDataRenderer take a value instead?

Reply:
I agree, mutating is not the best option here.

I suggest to change this statement in ListDataRenderer:

`const values = get(licensed, item, [])` and accept a property that is the value itself.

Reply:
I've removed the mutation by passing the value directly to the `ListDataRenderer` as suggested (via an optional prop)